src/filetree.rs

```diff
@@ -508,4 +595,127 @@ mod tests {
         assert!(!a.join(relp).join("shim.x64").exists());
         Ok(())
     }
+    #[test]
+    fn test_get_subdir() -> Result<()> {
+        // test path
+        let path = Utf8Path::new("foo/subdir/bar");
+        let (tp, tp_tmp) = get_subdir(path)?;
+        assert_eq!(tp.eq("foo"), tp_tmp.eq(".btmp.foo"));
```

Comment:
This seems odd - we're asserting on the results of two comparisons?
Did you mean to do
```
assert_eq!(tp, "foo");
assert_eq(tp_tmp, ".btmp.foo");
```
?

Reply:
Yes, you are correct, thanks!